code/game/turfs/space/transit.dm

```diff
@@ -90,6 +90,8 @@
 		return
 	if(istype(AM, /obj/docking_port) || !AM.simulated)
 		return //this was fucking hilarious, the docking ports were getting thrown to random Z-levels
+	if(istype(AM, /mob/dead))
+		return
 	var/max = world.maxx-TRANSITIONEDGE

```

Comment:
An undocumented (but appreciated!) feature. Just, please be sure to note that you changed this.